about/design-goals.md

```diff
@@ -0,0 +1,29 @@
+# CloudEvents Specification Design Goals
+
+CloudEvents are typically used in a distributed system to allow for services to be loosely coupled during development, then then configured with tight-coupling for specific applications.
```

Comment:
Typo "then then"?

Reply:
perhaps "rather than"  ?

Reply:
I was attempting to describe how late-binding of trigger + action allows for tight-binding when an application uses events. I believe that this is exactly what allows for observability.  Open to suggestions on how to clarify that!